src/pages/Login/index.tsx

```diff
@@ -13,17 +14,17 @@ const logo = require("../../resource/images/logo.png");
 
 export default () => {
   const navigate = useNavigate();
-  const { t } = useTranslation("login");
+  let { t } = useTranslation(["login", "error"]);

```

Comment:
Why do you use let instead of const ?

Reply:
I was testing some stuff and I forgot to revert it